class.jetpack.php

```diff
@@ -568,28 +568,20 @@ static function delete_active_modules() {
 	public function add_configure_hook() {
 		global $wp_filter;
 
-		$current_priority = has_filter( 'plugins_loaded', array( $this, 'configure' ) );
-		if ( false !== $current_priority ) {
-			remove_action( 'plugins_loaded', array( $this, 'configure' ), $current_priority );
+		if ( false === has_filter( 'plugins_loaded', [ $this, 'configure' ] ) ) {
+			add_action( 'plugins_loaded', [ $this, 'configure' ], - PHP_INT_MAX );
 		}
 
-		$taken_priorities = array_map( 'intval', array_keys( $wp_filter['plugins_loaded']->callbacks ) );
-		sort( $taken_priorities );
+		$our_action                = _wp_filter_build_unique_id( 'plugins_loaded', [ $this, 'configure' ], - PHP_INT_MAX );
```

Comment:
Should we be using WordPress private API? I like the idea of re-arranging callbacks, and I think it can be combined with Jetpack's current approach instead of using `PHP_INT_MAX`, but we can definitely do it using public API, can't we?

Reply:
How would you suggest to do this @zinigor? Removing all actions on the highest priority and re-adding it in the right order?

Reply:
@zinigor I pushed another version as you suggested, please check! We can probably improve it to avoid re-ordering the actions if it's not needed, but you get the idea.

BTW, I am off today and tomorrow, so I might not adjust the PR too fast...

Converning the use of WP public API: Well, a function is not private so I consider this as a public API (even if it's starts with a dash and it's not documented). And I don't think using the global `$wp_filter` and the public properties of `\WP_Hook` is much better. I believe this property has been left public for some kind of backward compatibility, but I don't see why it should be public. Anyway this is just my point of view, I am fine with both approaches.